### PR TITLE
Considering elterngeld only above minimum threshold for transfers

### PR DESCRIPTION
**Pull request by LauraGergeleit**
### What problem do you want to solve?

Referring to issue #520. So far, `elterngeld_m` is fully considered as income for most transfers. However,  `arbeitsl_geld` and `wohngeld` only consider the amount exceeding the minimum amount. For `arbeitsl_geld_2` (Bürgergeld) and `kinderzuschl` the full amount of `elterngeld_m` is considered as income but there is an exception if the parents still worked before the birth of the child, then they also receive an allowance of maximum 300 euros. Additionally, the multiple bonus of 300 euros per multiple child or 75 euros per sibling is not counted towards income for all transfers. 

Source: https://www.kindergeld.org/elterngeld-einkommen/

### Todo

- [x] Correct consideration of `elterngeld` for `wohngeld`
- [x] Adjust consideration of elterngeld for grunds_im_alter for multiple children

### CODE REVIEW

File: src/_gettsim/parameters/elterngeld.yaml [hunk 1]
```diff
@@ -172,3 +172,21 @@ max_monate_ind:
   2007-01-01:
     scalar: 12
     reference: Art. 1 G. v. 05.12.2006 BGBl. I S. 2748
+elterngeld_anr_frei:
```
Comment (ChristianZimpelmann) [inline src/_gettsim/parameters/elterngeld.yaml:175]: Cannot you just use the parameter `mindestbetrag`?

File: src/_gettsim/transfers/elterngeld.py [hunk 2]
```diff
@@ -453,3 +453,35 @@ def elterngeld_mehrlinge_bonus_m(
...
+
+    Returns
+    -------
+
+    """
+    out = max(
+        elterngeld_m
+        - (
+            1
+            + anz_mehrlinge_jüngstes_kind_hh * elterngeld_params["elterngeld_anr_frei"]
```
Comment (ChristianZimpelmann) [inline src/_gettsim/transfers/elterngeld.py:483]: is there a bracket missing around `1 + anz_mehrlinge_jüngstes_kind_hh`?

**Review by ChristianZimpelmann — COMMENTED** [review]
Thanks! I have two more comments.

File: src/_gettsim/parameters/elterngeld.yaml (hunk 1 above)
Comment (LauraGergeleit) [reply src/_gettsim/parameters/elterngeld.yaml:175]: Yes, I used the parameter which was in `grunds_im_alter` before but I guess the other solution is more elegant, I will change it

**Review by hmgaudecker — APPROVED** [review]
Looks great! 

Just one thing I am a little surprised about: Does this really affect only `grunds_im_alter` and `wohngeld`, but not `arbeitsl_geld_2` or `kinderzuschlag`? Seems like a somewhat strange selection at first glance, but I don't know the details well.

File: src/_gettsim/transfers/elterngeld.py [hunk 3]
```diff
@@ -453,3 +453,32 @@ def elterngeld_mehrlinge_bonus_m(
...
+    elterngeld_params
+        See params documentation :ref:`elterngeld_params <elterngeld_params>`.
+    anz_mehrlinge_jüngstes_kind_hh
+        See :func:`anz_mehrlinge_jüngstes_kind_hh`.
+
+    Returns
+    -------
+
+    """
+    mindestbetrag = elterngeld_params.get("mindestbetrag", 0.0)
```
[the remark this answers is not in the document]
Comment (LauraGergeleit) [reply src/_gettsim/transfers/elterngeld.py:479]: Okay, shall that be changed then? However, the tests for wohngeld in the year 2006 will fail then until its implemented

**Comment by LauraGergeleit** [comment]
> Just one thing I am a little surprised about: Does this really affect only `grunds_im_alter` and `wohngeld`, but not `arbeitsl_geld_2` or `kinderzuschlag`? Seems like a somewhat strange selection at first glance, but I don't know the details well.

I found [this source](https://www.kindergeld.org/elterngeld-einkommen/#Elterngeld_und_Buergergeld_frueher_Hartz_IV) that states that for `arbeitsl_geld_2` (Bürgergeld) and `kinderzuschl` the full amount of elterngeld_m is considered as income. There is an exception in which parents are granted an allowance of maximum 300 euros, if they still worked before the birth of the child. Not sure whether we want to implement these exceptions (at least in this PR)?

File: src/_gettsim/transfers/elterngeld.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (hmgaudecker) [reply src/_gettsim/transfers/elterngeld.py:479]: > the tests for wohngeld in the year 2006 will fail then until its implemented

Please mark the tests to `xfail`.

[the remark this answers is not in the document]
Comment (LauraGergeleit) [reply src/_gettsim/transfers/elterngeld.py:479]: If I remember correctly from another issue we were working on, it is quite tedious and messy to mark only the tests of one year to fail. Would it maybe be better to just comment the year out and write a comment above the code?

[the remark this answers is not in the document]
Comment (hmgaudecker) [reply src/_gettsim/transfers/elterngeld.py:479]: Just comment out all those tests :scream: for now, I have already gotten started working on #536, won't be long until that's implemented. 

It really seems time for #282 :smile:

**Comment by hmgaudecker** [comment]
@LauraGergeleit, can you fix the wohngeld test data and then it will be ready to merge? Apologies I missed this one before looking at #551.

**Pull request merged**